optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/db/os/OptimizeOpenSearchClient.java

```diff
@@ -368,6 +369,15 @@ public Set<String> getAllIndicesForAlias(final String aliasName) {
     }
   }
 
+  @Override
+  public boolean exists(final String indexName) throws IOException {
+    final ExistsRequest existsRequest =
+        new ExistsRequest.Builder()
+            .index(indexNameService.getOptimizeIndexAliasForIndex(indexName))
```

Comment:
does this get prefixed? And does it need to be?

Reply:
This gets prefixed if its not yet prefixed via getOptimizeIndexAliasForIndex, I wanted to mirror the ES logic here so that the equivalent ES/OS methods can use the same indexName parameter.